### PR TITLE
fix: ci

### LINKED ISSUE #219: [Omni-Bridge]: Fix CI

### PR DESCRIPTION
**Pull request by frolvanya**
closes #219

### CODE REVIEW

**Comment by frolvanya** [comment]
@karim-en If I remember correctly during the call you weren't happy about `wormhole-omni-prover-proxy` naming so I decided to remove it in this PR (https://github.com/Near-One/omni-bridge/pull/163/commits/2aaffc72f35ba14774f323859c57790922aa2165). Let me know if this is a bad idea, I'll revert commit

File: near/omni-types/src/prover_result.rs [hunk 1]
```diff
@@ -62,3 +62,14 @@ pub enum ProofKind {
     DeployToken,
     LogMetadata,
 }
+
+impl From<ProofKind> for u8 {
```
Comment (kiseln) [inline near/omni-types/src/prover_result.rs:66]: Could we use https://docs.rs/num_enum/latest/num_enum/ here?

Comment (frolvanya) [reply near/omni-types/src/prover_result.rs:66]: I tried using it, but I got a borsh error. Let me try again

**Review by kiseln — APPROVED** [review]

File: near/omni-types/src/prover_result.rs (hunk 1 above)
Comment (frolvanya) [reply near/omni-types/src/prover_result.rs]: Not sure where did I mess up the last time, but it works now: https://github.com/Near-One/omni-bridge/pull/163/commits/8601b573c0d1b0615ef9a58f4073df5ae4e00e2f

**Review by karim-en — APPROVED** [review]
